hack/update-codegen.sh

```diff
@@ -43,6 +44,16 @@ GOPATH=${WORK_DIR}/go/ go install k8s.io/code-generator/cmd/client-gen
 GOPATH=${WORK_DIR}/go/ go install k8s.io/code-generator/cmd/deepcopy-gen
 popd
 
+git clone --branch v0.2.4 --depth 1 -c advice.detachedHead=false \
```

Comment:
will this fail on second invocation?

Reply:
No. Each invocation uses a new `${WORK_DIR}`.